actors/providers/spring/src/main/java/com/ea/orbit/actors/providers/spring/SpringActorClassFinder.java

```diff
@@ -75,7 +78,9 @@ public void initialize()
         {
             throw new NullPointerException("orbit.actors.basePackage must be set!");
         }
-        this.concreteImplementations = actorImplementationProvider.getActorImplementations(actorInterfaceProvider.getActorInterfaces());
+        Map<Class<?>, Class<?>> implementations = actorImplementationProvider.getActorImplementations(actorInterfaceProvider.getActorInterfaces());
+        implementations.put(IReminderController.class, ReminderController.class); // built in Orbit actor
```

Comment:
Indeed. The reminder controller initialization needs to be done from the nodes. 
Thanks.